pkg/ble/configmap/type.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2020 The KubeEdge Authors.
```

Comment:
Refine the year. Same about other files.

Reply:
/lgtm, it had been fixed